src/components/Preevent.astro

```diff
@@ -0,0 +1,21 @@
+<section id="preevent" class="scroll-mt-20">
```

Comment:
I wouldn't create a new section for this, as it is part of the "About" section and doesn't need a scrollable anchor tag.